can-stream_test.js

```diff
@@ -1,17 +1,39 @@
 var QUnit = require('steal-qunit');
-var canStreamKefir = require('can-stream-kefir');
 var compute = require('can-compute');
 var DefineMap = require('can-define/map/map');
 var DefineList = require('can-define/list/list');
-
 var canStream = require('can-stream');
-// var canStream = canStream2(canStreamKefir);
 
 QUnit.module('can-stream');
 
 test('Compute changes can be streamed', function () {
 	var c = compute(0);
-	var stream = canStream.toStreamFromCompute(c);
+	var obj;
+	var canStreaming;
+
+
+
+	var canStreamInterface = {
```

Comment:
isn't this getting the `API` swapped with the `interface`?  From https://github.com/canjs/can-stream-kefir/issues/24, an interface should just provide:

```
toStream(compute):stream
toCompute(makeStream(setStream), context):compute
```